.github/workflows/stale.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v2.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # See https://github.com/actions/stale/blob/master/action.yml for description of the options
+
+          stale-issue-label: unloved
```

Comment:
We have a few labels with [Status] prefix, might work here too?

"[Status] unloved" would be quite a combo of serious and less serious, haha!

Reply:
I kinda feel those `[Status]` ones are for humans to mark... _unloved_ is quite short and sweet...

Reply:
Works for me 👍